data/brands/amenity/cinema.json

```
@@ -41,7 +41,7 @@
       }
     },
     {
-      "displayName": "AMC",
+      "displayName":",
```

Looks like you accidentally removed the displayName from AMC
```suggestion
      "displayName": "AMC",
```